# Inverse Kinematics input
string motor_group
bool only_pos
geometry_msgs/Point position
geometry_msgs/Point orientation
---
# Inverse Kinematics output
sensor_msgs/JointState joint_state
bool error